draft-kazuho-httpbis-priority.md

```diff
@@ -201,14 +201,25 @@ However, endpoints SHOULD continue sending end-to-end signals (e.g., the
 Priority header field), as that might have meaningful effect to other nodes that
 handle the HTTP message.
 
-An 8 bit value of 1 in HTTP/2 indicates support for HTTP/2 priorities
-as defined in Section 5.3 of [RFC7540] and is an error in HTTP/3 because
-there is not a clean mapping to HTTP/3.
+## Defined Prioritization Scheme Values
+
+This document defines two prioritization scheme values for use with the
+SETTINGS_PRIORITIES setting.
 
-## Negotiating the Extensible Priority Scheme {#settings-this-scheme}
+### H2_TREE {#settings-h2-scheme}
 
-The extensible priority scheme is negotiated using the described mechanism. It is
-identified by the 8-bit value of 2.
+This document defines the priority scheme identifier H2_TREE (8-bit value of 1)
+that indicates support for HTTP/2-style priorities ({{!RFC7540}}, Section 5.3).
+
+The H2_TREE priority scheme identifier MUST NOT be be sent in an HTTP/3 settings
+because there is no defined mapping of this scheme. Endpoints MUST treat receipt
+of H2_TREE as a connection error of type H3_SETTINGS_ERROR.
+
+### EXT_PRI {#settings-this-scheme}
```

Comment:
I'd call this EXTENSIBLE.  Including _PRI is redundant to the SETTING name.

I also don't think this is the name I'd choose, because if you extended the current scheme to add a new field, I believe that needs its own value?

Reply:
Then maybe ABSOLUTE?

Regardless, I think we can leave it as-is (or whatever) because it's does not affect the logic.

Reply:
Good points. If we applied @ianswett change on #80 the scheme would be called "Urgency Prioritization", so how about `URGENCY` as a replacement for `EXT_PRI`?

Reply:
Works for me.